### DISCUSSION TITLE
Is this picture real?

### DISCUSSION THREAD

**Discussion by Arashvscode** [Sketchware-Pro/Sketchware-Pro#547] [Q&A]
![Screenshot_20220826-110324_Rubika](https://user-images.githubusercontent.com/101287880/186837721-8ced749a-61ac-438c-b254-9dcdd689199e.jpg)

Hello, I wanted to see if this photo is real. This photo was published today by one of the Iranian copycat and thief channels

**Comment by rexluciano** [comment]
It's from other modders.

**Comment by j-px** [comment]
Seems real-ish, but in any way, it's not made by us Sketchware Pro Main modders. Let's at least hope they'll open up their patches, maybe when they're done.

**Reply by Arashvscode** [reply]
I think there is no need to add the grid in sk pro. With this, you have to make general changes, such as removing the local libraries section and many other things.
